R/dwdData.R

```diff
@@ -0,0 +1,204 @@
+################################################################################
+#' Download, unzip, untar data from an url
+#'
+#' This function download and untar and/or unzip dataset using url link from available dataset. Prior to download, the function
+#' can check if the files are already locally by using the checksum value available in dbHash.sqlite database.
+#'
+#' @param urltble A data.table that stores available dataset name, url, password and filenames found within each dataset.
```

Comment:
I suggest renaming the parameters to be more consistent with names in R, *e.g.*, `urlTbl`.